client/coral-framework/services/i18n.js

```diff
@@ -18,21 +19,21 @@ import daTA from 'timeago.js/locales/da';
 import deTA from 'timeago.js/locales/de';
 import esTA from 'timeago.js/locales/es';
 import frTA from 'timeago.js/locales/fr';
+import nl from 'timeago.js/locales/nl';
```

Comment:
Should this be `nlTA`?

Reply:
I only sorted those entries. For consistency it might be better to rename it, yes.

Reply:
Yes! I've just commited the change.